CMakeLists.txt

```
@@ -2096,7 +2096,8 @@ target_link_libraries(mixxx-lib PRIVATE mixxx-proto)
 target_include_directories(mixxx-lib SYSTEM PUBLIC lib/rigtorp/SPSCQueue/include)
 
 # Qt
-find_package(Qt5
+find_package(QT NAMES Qt5 COMPONENTS Core REQUIRED)
```

```suggestion
find_package(QT NAMES Qt6 COMPONENTS Core REQUIRED)
```
That is all that should be needed after this to test building with Qt6.